python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceTextGenerationParserPromptSchema.ts

```diff
@@ -17,15 +17,22 @@ export const HuggingFaceTextGenerationParserPromptSchema: PromptSchema = {
       temperature: {
         type: "number",
         minimum: 0,
-        maximum: 1,
+        maximum: 100,
+        description: `The temperature of the sampling operation. 
+        1 means regular sampling, 0 means always take the highest score, 
+        100.0 is getting closer to uniform probability.`,
```

Comment:
Is this a typo? Should it be 1.0?

Reply:
It should actually be 100 based on the docs https://huggingface.co/docs/api-inference/detailed_parameters